packages/@lwc/babel-plugin-component/src/__tests__/api-decorator.spec.js

```diff
@@ -735,6 +735,90 @@ describe('Transform property', () => {
             },
         }
     );
+
+    pluginTest(
+        '[W-9927596] - public property with duplicate getter/setter',
+        `
+        import { api } from 'lwc';
+        export default class Text {
+            @api foo = 1;
+            
+            get foo() {}
+            set foo(value) {}
+        }
+    `,
+        {
+            output: {
+                code: `
+                import { registerDecorators as _registerDecorators, registerComponent as _registerComponent } from "lwc";
+                import _tmpl from "./test.html";
+
+                class Text {
+                    foo = 1;
+
+                    get foo() {}
+
+                    set foo(value) {}
+                }
+                
+                _registerDecorators(Text, {
+                    publicProps: {
+                        foo: {
+                            config: 0,
+                        },
+                    },
+                });
+
+                export default _registerComponent(Text, {
+                    tmpl: _tmpl,
+                });
+              `,
+            },
+        }
+    );
+
+    pluginTest(
+        '[W-9927596] - public getter/setter with duplicate property',
+        `
+        import { api } from 'lwc';
+        export default class Text {
+            foo = 1;
+            
+            @api
+            get foo() {}
+            set foo(value) {}
```

Comment:
Is it worth testing the `@api set foo` case as well? What about the case where there is no `@api` on anything?

Reply:
I guess we know that these two cases are handled the same way internally, but we should add it in the spirit of black box testing.

Reply:
Add test for both use cases.